server/routes/incidentDetails/incidentDetailsValidation.ts

```diff
@@ -42,12 +42,17 @@ const errors: { [key: string]: FormError } = {
     href: '#incidentDate[time][minute]',
     text: 'Enter the minute using 2 numbers - for example, 08 or 18',
   },
+  FUTURE_TIME: {
+    href: '#incidentDate[time]',
+    text: 'Enter an incident time that is not in the future.',
+  },
 }
 
 export default function validateForm({
   incidentDate,
   locationId,
 }: incidentDetailsForm): FormError | FormError[] | null {
```

Comment:
Not a big deal - I'd tend to declare the variable in the code just before its used.